index.html

```diff
@@ -159,6 +159,51 @@
 
       <p>Some text goes here.</p>
     </section>
+
+
+    <section id="languages">
+      <h3><a href="#languages">Languages</a>
+      </h3>
+
+
+      <p>This document is focused on two languages: Standard Arabic and Persian.</p>
+
+
+      <section id="standard_arabic_language">
+        <h4><a href="#standard_arabic_language">Standard Arabic Language</a>
+        </h4>
+
+
+        <p><dfn>Standard Arabic</dfn>—a.k.a. Modern Standard Arabic or Literary Arabic—is the
+        standardized and literary variety of Arabic used in writing and in most formal speech in
+        countries of Northern Africa and West Asia. Regional and classical dialects of Arabic may
+        differ in layout and text details and are <em>not</em> covered by this document.</p>
+
+
+        <p>However, there are some major differences in common pratices between the <dfn>Western
+        Arab regions</dfn>—that is North-West Africa—and <dfn>Eastern Arab regions</dfn>—which is
+        North-East Africa and West Asia. For example, the numeral digits used in the two regions
+        and their formatting are vastly different. Although, there's no clear line between the
+        Eastern and Western Arab regions.</p>
+      </section>
+
+
+      <section id="persian_langauge">
+        <h3><a href="#persian_langauge">Persian Language</a>
+        </h3>
+
+
+        <p><dfn>Persian</dfn>—a.k.a. Modern Persian—is the standardized and literary variety of the
+        languages as used as official language in Iran and Afghanistan. The dialect of Persian is
```

Comment:
"the standardized and literary variety of the languages as used as official language in Iran and Afghanistan"
-> (?)
the standardized and literary variety of the official languages used in Iran and Afghanistan